mysqld_exporter.go

```diff
@@ -80,6 +92,21 @@ var (
 		"The total number of table I/O wait events for each table and operation.",
 		[]string{"schema", "name", "operation"}, nil,
 	)
+	performanceSchemaTableWaitsTimeDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, performanceSchema, "table_io_waits_picoseconds_total"),
+		"The total time of table I/O wait events for each table and operation.",
+		[]string{"schema", "name", "operation"}, nil,
+	)
+	performanceSchemaIndexWaitsDesc = prometheus.NewDesc(
```

Comment:
Yea, mysql is inconsistent in things.  Also chatting with @fabxc and @juliusv, I think I will convert all the time data to seconds if possible